apps/experiments/migrations/0036_update_AWS_synthetic_voices.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2 on 2023-10-11 15:18
+import os, json
+from django.db import migrations
+
+
+def load_language_code(apps, schema_editor):
+    SyntheticVoice = apps.get_model("experiments", "SyntheticVoice")
+    voice_data = {}
+    current_directory = os.path.dirname(os.path.abspath(__file__))
+    file_path = os.path.join(current_directory, "preload_data/aws_voices.json")
+
+    with open(file_path, "r") as json_file:
+        voice_data = json.load(json_file)["voices"]
+
+    voices_created = 0
+    voices_edited = 0
+    for voice in voice_data:
+        synth_voice, created = SyntheticVoice.objects.get_or_create(
+            name=voice["name"],
+            language=voice["language"],
+            gender=voice["gender"],
+            neural=voice["neural"],
+            service=voice["service"],
+        )
+
+        synth_voice.language_code=voice["language_code"]
+        synth_voice.save()
```

Comment:
You can do this more easily with 'update_or_create':
```suggestion
        synth_voice, created = SyntheticVoice.objects.update_or_create(
            name=voice["name"],
            gender=voice["gender"],
            neural=voice["neural"],
            defaults={
                "language": voice["language"],
                "service": voice["service"]
            }
        )
```